task/01-strings-tasks.js

```diff
@@ -69,7 +69,7 @@ function getStringFromTemplate(firstName, lastName) {
  *   'Hello, Chuck Norris!' => 'Chuck Norris'
  */
 function extractNameFromTemplate(value) {
-    throw new Error('Not implemented');
+    return value.substring(7, eval(value.length-1))
```

Comment:
what a reason to use `eval`? please don't use it
note substring can work with negative numbers also to take symbols from end. In the case `substring(7, -1);`